tools/lisa-build-asset

```diff
@@ -57,25 +73,119 @@ def list_assets(recipe_dir):
         if script.endswith('.recipe')
     )
 
+def get_recipe_name(asset):
+    return f'{asset}.recipe'
+
 def get_asset_recipe(asset, recipe_dir):
-    return os.path.join(recipe_dir, '{}.recipe'.format(asset))
+    return os.path.join(recipe_dir, get_recipe_name(asset))
 
-def make(asset, actions, arch, toolchain, recipe_dir, build_dir):
+def get_asset_property(asset, arch, property, **kwargs):
+    # Read asset's recipe to get a property value.
+    # Raise a ValueError if property is not set.
+    action = f"if [ -z ${{{property}+x}} ]; then exit 1; else printf '%s\\n' ${{{property}[@]}}; fi"
+    try:
+        val = make(asset, arch, [action], **kwargs, get_output=True)
+    except subprocess.CalledProcessError:
+        raise ValueError(f'The property "{property}" is not set.')
+    return val[action].splitlines()
+
+@contextmanager
+def make_chroot(asset, arch, build_dir=None, recipe_dir=None, arch_chroot_dir=None):
```

Comment:
`arch_chroot_dir=None` does not seem to be valid, it looks like this parameter should not have any default value. Same for all the other *_dir parameters, it seems the function won't work if it's not given all of them

Reply:
I did this to make it easier to identify the parameters (as requested in a previous comment)